Nef_3/test/Nef_3/test_nef_3_io_Cartesian_Lazy.cpp

```diff
@@ -23,8 +23,8 @@ void test_write_read()
   typedef CGAL::Polyhedron_3< Kernel >     Polyhedron;
   typedef typename Kernel::Point_3         Point;
 
-  typename Kernel::RT n(RT(std::string("6369051672525773")));
-  typename Kernel::RT d(RT(std::string("4503599627370496")));
+  typename Kernel::RT n(FT(std::string("6369051672525773")));
+  typename Kernel::RT d(FT(std::string("4503599627370496")));
```

Comment:
It is confusing (that's why the wrong code is currently there), but I believe the patch is right. `Kernel::RT` is in this case `Lazy_exact_nt<FT>` (the kernel is Cartesian or Simple_cartesian of Lazy_FT), while RT is something almost unrelated.

Reply:
Then should not we try to fix the problem: `Kernel::RT` being wrong?

Reply:
Uh, you want `Simple_cartesian<NT>::RT` to be different from NT? That would be strange.
In the testcase, we should remove RT and Lazy_RT, which are unused (probably this used to test the homogeneous kernel as well?).
Would you find it less confusing if I wrote something like `typename Kernel::RT n(typename Kernel::RT::Exact_type(std::string("6369051672525773")));` instead? IIRC the issue we are trying to solve is that converting directly from string to `Lazy_exact_nt<Q>` fails (can't remember why, I'll have to try again) and we need to give it a hint that it should first convert to Q.

Reply:
I did not understand the situation correctly. You are right that the types `RT` and `Lazy_RT` should be removed, to avoid confusion.

And I wonder why those lines use `Kernel::RT` when `Kernel::RT` is already a field type. Should not it be `Kernel::FT`, again to avoid the confusion? You are right that this probably used to support the homonegous case.